frontend/src/ErrorBoundaryDropParams.jsx

```diff
@@ -0,0 +1,32 @@
+import React from "react";
+
+export default class ErrorBoundary extends React.Component {
+  constructor(props) {
+    super(props);
+    this.state = { hasError: false };
+  }
+
+  static getDerivedStateFromError(error) {
+    // Update state so the next render will show the fallback UI.
+    return { hasError: true };
+  }
+
+  componentDidCatch(error, info) {
+    // Example "componentStack":
```

Comment:
Is this comment needed?

Reply:
good catch; this was straight from the react docs.